### PR TITLE
Replace non-inclusive term

### PR DESCRIPTION
**Pull request by SophieGuo410**
Remove part of non-inclusive term for close source backward compatible.

### CODE REVIEW

File: ambry-api/src/main/java/com/github/ambry/account/Container.java [hunk 1]
```diff
@@ -78,7 +78,7 @@
   static final String SECURE_PATH_REQUIRED_KEY = "securePathRequired";
   static final String OVERRIDE_ACCOUNT_ACL_KEY = "overrideAccountAcl";
   static final String NAMED_BLOB_MODE_KEY = "namedBlobMode";
-  static final String CONTENT_TYPE_WHITELIST_FOR_FILENAMES_ON_DOWNLOAD = "contentTypeWhitelistForFilenamesOnDownload";
+  static final String CONTENT_TYPE_ALLOW_LIST_FOR_FILENAMES_ON_DOWNLOAD = "contentTypeAllowListForFilenamesOnDownload";
```
[the remark this answers is not in the document]
Comment (SophieGuo410) [reply ambry-api/src/main/java/com/github/ambry/account/Container.java]: Discussed offline and confirmed with HI team, we only need to replace/annotate those terms which are locally defined in AmbryLI. so for this pr, I'll exclude the backward incompatible changes in open source.

File: ambry-api/src/main/java/com/github/ambry/account/ContainerBuilder.java [hunk 2]
```diff
@@ -248,8 +248,8 @@ public ContainerBuilder setReplicationPolicy(String replicationPolicy) {
   }
 
   /**
-   * Sets the whitelist for the content types for which filenames can be sent on download
-   * @param contentTypeWhitelistForFilenamesOnDownload the whitelist for the content types for which filenames can be
+   * Sets the allow list for the content types for which filenames can be sent on download
+   * @param contentTypeWhitelistForFilenamesOnDownload the allow list for the content types for which filenames can be
```
Comment (justinlin-linkedin) [inline ambry-api/src/main/java/com/github/ambry/account/ContainerBuilder.java:252]: should we also consider replace the variable name?

Comment (SophieGuo410) [reply ambry-api/src/main/java/com/github/ambry/account/ContainerBuilder.java]: My bad, revert the changes for the comments. will modify the name for the future pr.

**Review by justinlin-linkedin — APPROVED** [review]

File: ambry-api/src/main/java/com/github/ambry/config/NettyConfig.java [hunk 3]
```diff
@@ -183,8 +183,8 @@ public NettyConfig(VerifiableProperties verifiableProperties) {
     nettyServerRequestBufferWatermark =
         verifiableProperties.getIntInRange(NETTY_SERVER_REQUEST_BUFFER_WATERMARK, 32 * 1024 * 1024, 1,
             Integer.MAX_VALUE);
-    nettyBlacklistedQueryParams = new HashSet<>(
-        Arrays.asList(verifiableProperties.getString(NETTY_SERVER_BLACKLISTED_QUERY_PARAMS, "").split(",")));
+    nettyServerDenyListedQueryParams = new HashSet<>(
```
Comment (cgtz) [inline ambry-api/src/main/java/com/github/ambry/config/NettyConfig.java:186]: When you bring this change in, please update the closed source configs as well.

**Review by cgtz — APPROVED** [review]

**Pull request merged**